plugins/modules/rudder_node_settings.py

```diff
@@ -1,206 +1,57 @@
 #!/usr/bin/python
-# -*- coding: utf-8 -*-
-#
-# Copyright: (c) 2021, Rudder
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
 from __future__ import absolute_import, division, print_function
 
 import json
-
+import requests
 from ansible.module_utils.basic import AnsibleModule
 
-# Check if 'requests' is installed
-# Use : 'pip3 install requests' if not
-try:
-    import requests
-    from requests.packages.urllib3.exceptions import InsecureRequestWarning
-except ImportError:
-    HAS_REQUESTS = False
-else:
-    HAS_REQUESTS = True
-
 __metaclass__ = type
 
-# Disable SSL certificate warning messages
-requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
-
-DOCUMENTATION = """
----
-module: rudder_node_settings
-author:
-  - Rudder
-version_added: '2.9'
-short_description: Configure Rudder Nodes parameters via APIs
-requirements:
-    - 'python >= 2.7'
-
-options:
-
-  rudder_url:
-    description:
-      - Providing Rudder server IP address. Defaults to localhost of the target node if not set, with certificate validation disabled, unless explicitly enabled by setting validate_certs.
-    required: false
-    type: str
-
-  rudder_token:
```

Comment:
Don't forget to add the documentation, otherwise the code may be difficult to understand.